SliceTracker/SliceTracker.py

```diff
@@ -3576,3 +3579,143 @@ def onCaseNumberChanged(self, caseNumber):
+    self._showLegend = False
+    self.xAxis.SetTitle('Time Unit')
+    self.yAxis.SetTitle('Displacement')
+    
+    
+    self.chartTable = vtk.vtkTable()
+
+    self.arrX = vtk.vtkFloatArray()
+    self.arrX.SetName('X Axis')
+    
+    self.arrXD = vtk.vtkFloatArray()
+    self.arrXD.SetName('X Component')
+    
+    self.arrYD = vtk.vtkFloatArray()
+    self.arrYD.SetName('Y Component')
+    
+    self.arrZD = vtk.vtkFloatArray()
+    self.arrZD.SetName('Z Component')
+
+    self.arrX.InsertNextValue(0)
+    self.arrXD.InsertNextValue(0)
+    self.arrYD.InsertNextValue(0)
+    self.arrZD.InsertNextValue(0)
+
+    self.chartPopupWindow = None
+    self.chartPopupSize = qt.QSize(600, 300)
+    self.chartPopupPosition = qt.QPoint(0,0)
+    
+    self.showLegendCheckBox = qt.QCheckBox('Show legend')
+    self.showLegendCheckBox.setChecked(0)
+
+    self.plotFrame = ctk.ctkCollapsibleButton()
+    self.plotFrame.text = "Plotting"
+    self.plotFrame.collapsed = 0
+    plotFrameLayout = qt.QGridLayout(self.plotFrame)
+    self.plottingFrameWidget = qt.QWidget()
+    self.plottingFrameLayout = qt.QGridLayout()
+    self.plottingFrameWidget.setLayout(self.plottingFrameLayout)
+    self.plottingFrameLayout.addWidget(self.showLegendCheckBox)
+    self.plottingFrameLayout.addWidget(self._chartView)
+
+    self.popupChartButton = qt.QPushButton("Undock chart")
+    self.popupChartButton.setCheckable(True)
+    self.plottingFrameLayout.addWidget(self.popupChartButton)
+
+    plotFrameLayout.addWidget(self.plottingFrameWidget)
+
+    self.popupChartButton.connect('toggled(bool)', self.onDockChartViewToggled)
+    self.showLegendCheckBox.connect('stateChanged(int)', self.onShowLegendChanged)
+
+  def addPlotPoints(self,triplets):
+    self.chartTable.SetNumberOfRows(len(triplets))
+    for i in range(0,len(triplets)):
+      self.arrX.InsertNextValue(i+1)
+      self.arrXD.InsertNextValue(triplets[i][0])
+      self.arrYD.InsertNextValue(triplets[i][1])
+      self.arrZD.InsertNextValue(triplets[i][2])
+
+    self.chartTable.AddColumn(self.arrX)
+    self.chartTable.AddColumn(self.arrXD)
+    self.chartTable.AddColumn(self.arrYD)
+    self.chartTable.AddColumn(self.arrZD)
+    
+    plot = self.chart.AddPlot(vtk.vtkChart.LINE)
+    plot.SetInputData(self.chartTable,0,1)
+    plot.SetColor(255,0,0,255)
+    
+    plot2 = self.chart.AddPlot(vtk.vtkChart.LINE)
+    plot2.SetInputData(self.chartTable,0,2)
+    plot2.SetColor(0,255,0,255)
+    
+    plot3 = self.chart.AddPlot(vtk.vtkChart.LINE)
+    plot3.SetInputData(self.chartTable,0,3)
+    plot3.SetColor(0,0,255,255)
+
+
+  def onShowLegendChanged(self, checked):
+    if checked == 2:
+      self.chart.SetShowLegend(True)
+    else:
+      self.chart.SetShowLegend(False)
+
+  def onDockChartViewToggled(self,checked):
+    if checked:
+      self.chartPopupWindow = qt.QDialog()
+      self.chartPopupWindow.setWindowFlags(PythonQt.QtCore.Qt.WindowStaysOnTopHint)
```

Comment:
@Kmacneil0102 You don't need to explicitly import PythonQt. Just use qt.Qt.WindowStaysOnTopHint